build-tools/src/main/resources/repose-style-configs/scalastyle_config.xml

```diff
@@ -93,7 +93,8 @@
         <customMessage>Require space after comma, semicolon, and colon not in string</customMessage>
     </check>
 
-    <check enabled="true" class="org.scalastyle.file.RegexChecker" level="warning">
+    <!-- this causes too many false positives -->
+    <check enabled="false" class="org.scalastyle.file.RegexChecker" level="warning">
```

Comment:
We could make it slightly stricter and disallow '{' being on a line of its own. Is there a case where that's valuable?
`foo {
    doSomething()
    doSomethingElse()
}`
vs
`foo
{
    doSomething()
    doSomethingElse()
}`